bitnami/kafka/templates/log4j-configmap.yaml

```diff
@@ -12,5 +12,5 @@ metadata:
   {{- end }}
 data:
   log4j.properties: |-
-    {{ .Values.log4j | nindent 4 }}
+    {{ tpl .Values.log4j $ | nindent 4 }}
```

Comment:
Could we use our standard for tpl values as well, please?

```
{{- include "kafka.tplValue" ( dict "value" .Values.log4j "context" $ ) | nindent 4 }}
```

Reply:
done